packages/snap/src/filecoin/account.ts

```diff
@@ -1,12 +1,21 @@
 import {Wallet} from "../interfaces";
-import {KeyPair, keyPairFromSeed} from "@nodefactory/filecoin-address";
+// eslint-disable-next-line
+// @ts-ignore
+import {keyDeriveFromSeed} from "@zondax/filecoin-signing-tools/js";
```

Comment:
Why ts-ignore if you defined custom types?

Reply:
@mpetrun5 Actually I messed up here, as I should change `ts-config` file so it defines types for `@zondax/filecoin-signing-tools/js` and not for `@zondax/filecoin-signing-tools` and then I wouldn't need `@ts-ignore`.

But then I fixed it while doing the next task, so I suggest to leave it as it is because it will be fixed in the next PR.